src/main/java/com/diablo3CharViewer/data_models/ItemWeaponDataModel.java

```diff
@@ -1,24 +1,27 @@
 package com.diablo3CharViewer.data_models;
 
+import java.util.List;
+import java.util.Map;
+
 public class ItemWeaponDataModel extends ItemDataModel {
 
     private String minDamage;
     private String maxDamage;
```

Comment:
can be final?

Reply:
It is final now. Can be final because this field value is fetched once  from api response and then set to data model.